cloudvolume/datasource/precomputed/sharding.py

```diff
@@ -586,18 +589,42 @@ def get_data(self, label, path="", progress=None):
       for ext in exists.values()
     )
 
+    # Requesting many individual shard chunks is slow, but due to z-ordering
+    # we might be able to combine adjacent byte ranges. Especially helpful
+    # when downloading entire shards!
+    bundles = []
+    for chunk in sorted(files, key=itemgetter("path", "start")):
```

Comment:
`sorted` creates a new array vs `.sort` does it in-place. Would `.sort` be faster? Is it substantial compared to the rest?